website/packages/docs/src/pages/pkg-babel-plugin.mdx

```diff
@@ -93,6 +94,15 @@ Extensions that we should consider code. We use these to identify if a file shou
 - Type: `string[]`
 - Default: `['.js', '.jsx', '.ts', '.tsx']`
 
+#### hashPrefix
+
+Adds a prefix to the generated hashed css rule names.
```

Comment:
nit: mention what format `hashPrefix` is required to be

e.g. how it needs to match the `/^[a-zA-Z\-][a-zA-Z\-_0-9]*$/` regex

Reply:
ah yep good idea!